Saber/Renderer.cpp

```diff
@@ -364,8 +362,8 @@ void Renderer::PerformResize() {
         return;
 
     // Don't allow 0 size swap chain back buffers.
-    m_clientWidth = std::max(1u, width);
-    m_clientHeight = std::max(1u, height);
+    m_clientWidth  = std::min<uint32_t>(std::max(1u, width), 4096);
```

Comment:
so - no 8k gaming& leave as is now, later add second spd pass for this case, maybe 16k gaming will usethis engine too

Reply:
:( Also TODO in DepthBuffer::ResizeHZB